encoding/wkb/sql.go

```diff
@@ -18,6 +18,16 @@ func (e ErrExpectedByteSlice) Error() string {
 	return fmt.Sprintf("wkb: want []byte, got %T", e.Value)
 }
 
+// A Geom is a WKB-ecoded Geometry that implements the sql.Scanner and
+// driver.Value interfaces.
+// It can be used when the geometry shape is not defined.
+type Geom struct {
+	geom.T
+	// NOTE(tb) I've repeated the opts field like other types, but I'm not sure
+	// about its usage.
```

Comment:
These options control how the geometry is encoded, specifically how empty points are handled.

Reply:
ok thanks, I removed the note 
92b5cec